### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by hangyao**
Updated SystemB sink format

### CODE REVIEW

**Comment by hangyao** [comment]
Merged with all previous commits,

And updated System B sink format

**Comment by hangyao** [comment]
Fixed conflict with the latest updates on System B

File: A1-2015/SystemB/SinkFilter.java [hunk 1]
```diff
@@ -67,7 +72,7 @@ public void run()
 		try
 		{
 			outStream = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(fileName)));
-			outStream.writeUTF("Time:\t" + "Temperature (C):\t" + "Altitude (m):\t" + "Pressure (PSI):\t" + "\n");
+			outStream.writeUTF("Time:\t\t\t" + "Temperature (C):\t" + "Altitude (m):\t" + "Pressure (PSI):\t" + "\n");
```
Comment (darioalessandro) [inline A1-2015/SystemB/SinkFilter.java:75]: 👍

File: A1-2015/SystemA/Plumber.java [hunk 2]
```diff
@@ -1,3 +1,110 @@
...
+
+	    fileReaderSource.start();
+	    fahrenheitToCelsius.start();
+		feetToMeters.start();
+	    sink.start();
+
+   } // main
+
+} // Plumber
+=======
```
Comment (darioalessandro) [inline A1-2015/SystemA/Plumber.java]: I think this is some git conflict stuff, we probably need to take it out before merging.

File: A1-2015/SystemA/Plumber.java [hunk 3]
```diff
@@ -1,3 +1,110 @@
...
+
+	    fileReaderSource.start();
+	    fahrenheitToCelsius.start();
+		feetToMeters.start();
+	    sink.start();
+
+   } // main
+
+} // Plumber
+||||||| merged common ancestors
```
Comment (darioalessandro) [inline A1-2015/SystemA/Plumber.java]: I think that we need to comment this?

File: A1-2015/SystemA/Plumber.java [hunk 4]
```diff
@@ -1,3 +1,110 @@
+<<<<<<< HEAD
```
Comment (darioalessandro) [inline A1-2015/SystemA/Plumber.java]: I think that this is some merge conflict stuff.

**Review by darioalessandro — COMMENTED** [review]
Hey @hangyao I think that there are some merge conflicts you might to look at before merging, I'll take a look after you revise it.

File: A1-2015/SystemA/Plumber.java [hunk 5]
```diff
@@ -52,3 +159,4 @@ public static void main(String argv[]) {
         transformTemperatureAndConvertAltitude.start();
     }
 }
+>>>>>>> e50cb802bb3074141a2b791fb13c5e0d7ecdbac5
```
Comment (darioalessandro) [inline A1-2015/SystemA/Plumber.java]: merge conflict stuff.

**Comment by hangyao** [comment]
Revised and solved the conflicts.

**Review by darioalessandro — APPROVED** [review]
LGTM

**Pull request merged**